server/utils.go

```diff
@@ -223,3 +223,28 @@ func (s *Server) FilterDisallowedAnnotations(toFind, toFilter map[string]string,
 
```

Comment:
nit I'm personally a fan of keeping a function closest to the function that uses it, and reserving utils files for when the function is needed across multiple files. WDYT about moving this and the associated test?

Reply:
Moved from utils.go to container_create_linux.go